src/main/scala/org/holmesprocessing/totem/services/richheader/rich.py

```diff
@@ -63,6 +66,20 @@ def get(self):
         except Exception as e:
             self.write({"error": traceback.format_exc(e)})
 
+        try:
+            func_data = RichFunctionsRun(fullPath, rich_data, 4)
+        except richfuncfinder.MachineVersionError:
+            self.write({'error'}: richfuncfinder.err2str(-1))
+        except richfuncfinder.NoMatchingSignatures:
+            self.write({'error'}: richfuncfinder.err2str(-2))
+        except richfuncfinder.UnknownRelocationError:
+            self.write({'error'}: richfuncfinder.err2str(-3))
+        except Exception as e:
+            self.write({"error": traceback.format_exc(e)})
+
+        full_data = rich_data.append(func_data)
```

Comment:
Not sure if this will work correctly (appending JSON outputs). Maybe use "+" ? or: full_data = {'richheader': rich_data, richfunctions: func_data}. But this may break compatibility with already existing tools using this service !.